src/components/Header/main-logo.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import { CircularElement } from "~/components/footer/circular.tsx";
+import { LogoContainer, StyledTypography } from "~/components/footer/styles.ts";
+import { Link } from "~/components/header/link.tsx";
+import { colors } from "~/libs/constants/color.ts";
+
+export const MainLogo: React.FC = () => {
+	const { t } = useTranslation();
+	return (
```

Comment:
empty line above, we need some separation, to easier read

Reply:
Seems like linter/prettier deleted the space here. Check if that’s the case, and if so, let me know - i will fix the linter.
@yuliiaaaaa

Reply:
Seems like linter/prettier deleted the space here. Check if that’s the case, and if so, let me know - i will fix the linter. @yuliiaaaaa

Reply:
done